### DISCUSSION TITLE
Example.com hard-coded instead of referencing environment variable

### DISCUSSION THREAD

**Discussion by wabarkley** [htpcBeginner/docker-traefik#111] [Support]
This bit is sanitized with "example.com", but it's not readily apparent how to make it reference the configured DOMAINNAME environment variable, or if that's even what we're going for here. 

https://github.com/htpcBeginner/docker-traefik/blob/2017cc3c1e163772609687f866cea9e080344ebc/traefik2/rules/middlewares.toml.example#L38

Does this entry just get moved to the docker-compose file's traefik section under labels?

**Comment by SimpleHomelab** [comment]
As far as I know you cannot use environmental variables (DOMAINNAME) in toml/yml files. As explained in my guides, example.com will have to customized to your situation. 

Moving to discussions as this is not an issue.

**Reply by Flute7** [reply]
Should this be https://example.com if Organzir is available at www.example.com (redirect from example.com)?

**Reply by wabarkley** [reply]
When I went looking for the answer, I landed at https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options, which states that the "allow-from" directive is obsolete for modern browsers. The allowable choices are DENY and SAMEORIGIN, so perhaps this line isn't even relevant any more, or should be set to SAMEORIGIN instead? (Or replaced by setting a different header 'Content-Security-Policy' 'frame-ancestors' directive as described at https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors)
